client/rpcserver/handlers.go

```diff
@@ -739,4 +765,19 @@ Registration is complete after the fee transaction has been confirmed.`,
 		returns: `Returns:
     string: The message "` + fmt.Sprintf(canceledOrderStr, "[order ID]") + `"`,
 	},
+	withdrawRoute: {
+		pwArgsShort: `"appPass"`,
+		argsShort:   `assetID value "address"`,
+		cmdSummary:  `Withdraw value from an exchange wallet to address.`,
+		pwArgsLong: `Password Args:
+    appPass (string): The DEX client password.`,
+		argsLong: `Args:
+    assetID (int): The asset's BIP-44 registered coin index. Used to identify
+      which wallet to withdraw from. e.g. 42 for DCR. See
+      https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+    value (int): The amount in the asset's smallest unit to withdraw.
```

Comment:
Nit: "The amount to withdraw in units of the asset's smallest denomination (e.g. satoshis, atoms, etc.)"